postgres_exporter/common/pg15/setup.sql

```diff
@@ -0,0 +1,463 @@
+-- PG15 pgMonitor Setup
+--
+-- Copyright © 2017-2022 Crunchy Data Solutions, Inc. All Rights Reserved.
+--
+
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = 'ccp_monitoring') THEN
+        CREATE ROLE ccp_monitoring WITH LOGIN;
+    END IF;
+
+    -- The pgmonitor role is required by the pgnodemx extension in PostgreSQL versions 9.5 and 9.6
+    -- and should be removed when upgrading to PostgreSQL 10 and above.
+    IF EXISTS (SELECT 1 FROM pg_roles WHERE rolname = 'pgmonitor') THEN
+        DROP ROLE pgmonitor;
+    END IF;
```

Comment:
This is if you're upgrading from an old version of Pg? not sure how/why that would be the case given that we're using Pg 15 now, but I assume it's something like 1) upgrade pg from 9.5 or 9.6 to PG 15, 2) install pgmonitor. This correct, or is this actually cruft?

Reply:
So on the off chance that someone set up pgmonitor back on 9.5 or 9.6 and are now upgrading to 10 or later, then this would still need to be there. Would need to get a survey of all clients and what they're currently running before I would want to remove this.